### DISCUSSION TITLE
Project Management process with issues and merge requests

### DISCUSSION THREAD

**Discussion by L-U-C-K-Y** [community/community#47750] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi all

We have switched to GitHub projects for project management and I would love to have your feedback on how you're managing your workflow with it.

For example, here is how we do it, but I think it's not so ideal:

**Assuming we have the following sections on the board:**

1. Backlog
2. Ready
3. In Progress
4. Tech Review
5. QA Review
6. Done

**Description:**

1. Feedback comes in the **backlog** without any issue created for it
2. We then analyze, describe, priotitize, create an issue in a repo and put it into the read column. If there are questions before starting, we have everything in the issue
3. A dev, then takes the issue and moves it to in progress, creates an MR and references the issue. **QUESTION:** Now we have an MR with a very similar title on the board and the issue. **Q1:** Is it correct to have 1 issue and 1 MR on the board? 
4. When finishing, we put the issue it into tech review. **QUESTION:** **Q2:** Where should we gather technical feedback? I guess in the MR?
5. When the tech review is finished, our QA comes and checks the issue.  **QUESTION:** **Q3** We now have the QA feedback again in the Issue, not the MR, still 2 elements on the board, is this ok?
6. When QA finds no issues, we move the issue to done **QUESTION:** **Q4** Should the QA merge the MR at the end?

**Questions:**

1. Is it correct to have 1 issue and 1 MR on the board? 
2. Where should we gather technical feedback? I guess in the MR?
3. We now have the QA feedback again in the Issue, not the MR, still 2 elements on the board, is this ok?
4. Should the QA merge the MR at the end?

Looking forward to your feedback?

Kind regards

Lucky

**Comment by michaelmccallam** [comment]
Hi L-U-C-K-Y,

1. Is it correct to have 1 issue and 1 MR on the board?
// It's easier to just manage the issues on the Kanban board, this includes draft issues

3. Where should we gather technical feedback? I guess in the MR?
// Best to keep all discussion in code review comments - if detailed docs are required or links to Notion etc, they can be kept on the issues.
5. We now have the QA feedback again in the Issue, not the MR, still 2 elements on the board, is this ok?
//Keep the QA feedback in code review comments on the PR
6. Should the QA merge the MR at the end?
// best for who ever created the PR to merge it, regardless of the roles. If an engineer is merging a branch with lots of tests added by QA, then it's best to keep ownership for who ever is leading and owning that feature - this is to avoid a ' throw it over the fence to QA' situation.
